scene/main/viewport.cpp

```diff
@@ -1574,11 +1561,6 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 				_gui_call_input(mouse_focus, mb);
 			}
 
-			/*if (gui.drag_data.get_type()!=Variant::NIL && mb->get_button_index()==MOUSE_BUTTON_LEFT) {
-				_propagate_viewport_notification(this,NOTIFICATION_DRAG_END);
-				gui.drag_data=Variant(); //always clear
-			}*/
-
```

Comment:
Commented out 5 years ago
in https://github.com/godotengine/godot/commit/db9335f4c1247d950b4858b56d653975bb3058ff